packages/web/localizations/es.json

```diff
@@ -2,7 +2,7 @@
   "app": {
     "banner": {
       "title": "Estás viendo todos los activos sin permiso",
-      "linkText": "Estás viendo todos los activos sin permiso. <Enlace> Haga clic aquí para volver a la aplicación principal </link>."
+      "linkText": "Estás viendo todos los activos sin permiso. <link> Haga clic aquí para volver a la aplicación principal </link>."
```

Comment:
Do we want to change this? It seems to be markup to be processed later.

Reply:
"Enlace" is the translation of the word "link" in Spanish. If you check the English, French, Korean, etc translations, they have the word "link". It seems to be one of the several reasons why I think it's a google translation.